config/schedule.rb

```diff
@@ -29,6 +29,7 @@
 
 every 1.minute do
   rake "external_api_client:drive_cron_jobs"
+  rake "ahoy:fill_empty_location_entries"
```

Comment:
driving this every minute will be too expensive, drive it every 12 hours instead; https://github.com/restarone/violet_rails/blob/3ccb6076b162e052fbb35f5c848fd36286ce96c5/config/schedule.rb#L27

Reply:
correct. i did moved there for testing(crontab)